SIVA/Core/Files/Readers/Servers.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Discord.WebSocket;
+using SIVA.Core.Files.Objects;
+using Newtonsoft.Json;
+using SIVA.Core.Discord;
+using SIVA.Core.Runtime;
+
+namespace SIVA.Core.Files.Readers
+{
+    public static class ServerConfig
+    {
+        private static readonly List<Server> Config = new List<Server>();
+        private const string FilePath = "data/serverconfigs.json";
+
+        static ServerConfig()
+        {
+            try
+            {
+                var jText = File.ReadAllText(FilePath);
+                Config = JsonConvert.DeserializeObject<List<Server>>(jText);
```

Comment:
WOWZA! You should probably make a file for each server, otherwise that's gonna get messy.

Reply:
already is messy lol